content/docs/0.8.x/reference/bridge/manage_projects/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Manage Projects
+description: Create, update and deletes projects from the Bridge
+weight: 21
+keywords: [0.8.x-bridge]
+---
+
+The Bridge provides you with features to make managing projects more convenient. It is possible to create and update projects directly from the UI.
+The following sections explain in detail which functionalities you can use to set up your projects and where to find them.
+
+This may be also interesting for you:
+
+* [Create a project with CLI](https://keptn.sh/docs/0.8.x/reference/cli/commands/keptn_create_project/)
+* [Update a project with CLI](https://keptn.sh/docs/0.8.x/reference/cli/commands/keptn_update_project/)
+
+## Create a new project in Bridge
+
+Going to the project overview, you can find a button "Create a new project" there at the bottom of the page.
+Clicking on this button redirects you to a new page where you find a form that provides all the necessary information needed for creating a new project in Keptn.
```

Comment:
```suggestion
Going to the project overview, you will find a button "Create a new project" at the bottom of the page.
Clicking on this button redirects you to a new page where you will find a form that provides all the necessary information needed for creating a new project in Keptn.
```

Reply:
Simplified it:
`Going to the project overview, there is a button "Create a new project" at the bottom of the page.
Clicking on this button redirects you to a new page with a form that provides all the necessary information needed for creating a new project in Keptn.`